docs/ref/contrib/auth.txt

```diff
@@ -315,6 +315,14 @@ Manager methods
         Same as :meth:`create_user`, but sets :attr:`~models.User.is_staff` and
         :attr:`~models.User.is_superuser` to ``True``.
 
+    .. method:: with_perm(perm, obj=None)
+
+        .. versionadded:: 1.11
+
+        Returns a queryset containing :class:`~django.contrib.auth.models.User`
+        objects that have the given permission ``perm`` in the form of
+        ``"<app label>.<permission codename>"``, including superusers.
```

Comment:
Thanks, I sent an email to django-developers: https://groups.google.com/forum/#!topic/django-developers/WXgAN7X0qkY